.golangci.yml

```diff
@@ -6,26 +6,34 @@ run:
   - "zz_generated.*\\.go$"
   - ".*conversion.*\\.go$"
 linters:
+  disable-all: true
   enable:
+    - deadcode
+    - goconst
+    - gofmt
     - golint
+    - gosec
     - govet
-    - gofmt
-    - structcheck
-    - varcheck
-    - interfacer
-    - unconvert
     - ineffassign
-    - goconst
-    # - gocyclo
+    - interfacer
     - maligned
     - misspell
     - nakedret
     - prealloc
-    - gosec
-    - deadcode
-  disable-all: true
+    - structcheck
+    - unconvert
+    - varcheck
   # Run with --fast=false for more extensive checks
   fast: true
-issue:
+issues:
```

Comment:
The parser in `golangci-lint` seems to silently ignore typos like this (should have been [`issues:`](https://golangci-lint.run/usage/configuration/#config-file), so this section was being ignored.